src/docs/ftw-analytics/how-to-set-up-analytics-for-ftw/index.md

```diff
@@ -1,61 +1,138 @@
 ---
 title: How to set up Analytics for FTW
 slug: how-to-set-up-analytics-for-ftw
-updated: 2021-04-29
+updated: 2021-05-09
 category: ftw-analytics
 ingress:
   This guide describes how to set up analytics for Flex Template for Web
   (FTW).
 published: true
 ---
 
-FTW supports tracking page views with pluggable analytics handlers.
+FTW comes with built-in support for Google Analytics and supports
+tracking page views with customisable analytics handlers. This article
+explains how to enable Google Analytics and use and create custom
+analytics handlers.
 
-## Configure Google Analytics ID
+## Configure Google Analytics
 
-Google Analytics (GA) is supported by default. To enable GA, set the GA
-tracker ID to the environment variable `REACT_APP_GOOGLE_ANALYTICS_ID`;
+FTW has built-in support for Google Analytics. All you need to do is
+assign your Google Analytics Tracking ID to the environment variable
+`REACT_APP_GOOGLE_ANALYTICS_ID`.
 
-Note that the current script expects a _Universal Analytics property
-only_ tracking ID. If you want to use GA4, you should update the Google
-Analytics scripts first.
+> Google Analytics doesn't work in a hot-loading environment! The
+> analytics script is added server-side. You can test it in your local
+> environment by using the command `yarn run dev-server`.
 
-That's it! You're good to go.
+### Google Analytics 4
 
-> Note: Google Analytics doesn't work in hot-loading environment! <br>
-> GA script is added in server-side setup. You can test it local
-> environment with `yarn run dev-server`.
+Google recently released their new analytics service Google Analytics 4.
+Support for Google Universal Analytics will end on October 1, 2023. New
+versions of FTW provide out-of-the-box support for Google Analytics 4.
 
-## Optional: Use a custom analytics handler
+If you're starting development on a new version of FTW and prefer to use
+Universal Analytics, you should look into how Analytics was implemented
+before
+[this pull request](https://github.com/sharetribe/ftw-daily/pull/1508).
+If you have built your marketplace on top of an older version of FTW and
+you want to start using Google Analytics 4, you'll need to implement the
+changes introduced in
+[this pull request](https://github.com/sharetribe/ftw-daily/pull/1508).
```

Comment:
The repetition of "this pull request" sounds like there are two pull requests, one that introduces the changes and one before which you can check the Universal Analytics version. I don't know if it's a problem, so up to you whether you want to reformulate this!

Reply:
Good point, fixed!